content/components/banner.mdx

```diff
@@ -21,45 +21,30 @@ Banner can be used in one of two ways:
-
-On the other hand, for communicating newly shown non-critical information (such as a success), a live region announcement is sufficient in **most** scenarios. A live region announcement can sometimes fail to fire and can easily be missed, so it should be reserved for use on non-critical information where there are no consequences if a user misses it.
+#### Will the user be blocked if they miss the Banner?
 
-#### Where is a user's focus when the banner appears?
-
-If an action triggers updates to the UI resulting in a user's focus getting lost and a new Banner being shown, place focus on this newly shown Banner instead of using a live region announcement.
+If a user is blocked from proceeding if they miss the Banner, moving focus is usually preferred over a live region announcement.
 
-For example, let's say that a user activates a "Submit" button. Upon doing so, significant portions of the UI (including this button) gets removed and replaced by a success banner. The fact that this is a non-critical banner may lead us to think announcing the banner is sufficient. However, given that the user experiences focus loss as a result of the control they were on being removed from the page, focus management is more appropriate.
+A prime example of this is an error banner shown that appears after a form validation, such as [interactive error summary](https://primer.style/ui-patterns/forms/overview#interactive-summary-of-errors). Moving focus directly into the error banner allows all users (including screen reader users, zoom magnification users, and sighted keyboard users), to immediately perceive the feedback and take the necessary action to resolve the issue.
 
-This is because focus loss can result in some screen reader users being taken back to the top of the page and be disorienting. This is a [WCAG 2.4.3 Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html) issue.
+If the banner communicates non-critical information (e.g. a success banner) that wouldn't block a user if missed, a live region announcement is sufficient.
 
-#### Does the Banner contain an action?
-
-If the Banner contains an action, placing focus on the newly shown Banner is preferred over a live region announcement so that a screen reader user can immediately interact with the action without having to manually locate it.
+#### Where is a user's focus when the banner appears?
 
-#### Is there already a live region on the page that can be utilized?
+If a user triggers an action that results in the element that they were were on being removed from the DOM, then focus management is required. This is mostly relevant in SPA/React where elements in the DOM are replaced, resulting in focus loss.
```

Comment:
This paragraph sounds like it's covering where focus is placed when the banner is dismissed. It may not be, but it's a bit vague. We might need an example or a tie back into the section title here.

Reply:
Thanks Chelsea! I added an example, hope it helps!